sdk/python/distillation/benchmarking/pipelines/conversation/hellaswag.yaml

```diff
@@ -0,0 +1,286 @@
+  downloader_fewshot:
+    type: command
+    component: azureml://registries/azureml/components/dataset_downloader/labels/latest
+    limits: {}
+    inputs:
+      dataset_name: hellaswag
+      configuration: all
+      split: train
+    outputs:
+      output_dataset:
+        type: uri_folder
+  sampler_fewshot:
+    type: command
+    component: azureml://registries/azureml/components/dataset_sampler/labels/latest
+    limits: {}
+    inputs:
+      dataset:
+        type: uri_folder
+        path: ${{parent.jobs.downloader_fewshot.outputs.output_dataset}}
+      sampling_style: head
+      n_samples: 8
+      random_seed: 0
+    outputs:
+      output_dataset:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.jsonl
+  preprocessor_fewshot:
+    type: command
+    component: azureml://registries/azureml/components/dataset_preprocessor/labels/latest
+    limits: {}
+    inputs:
+      dataset:
+        type: uri_folder
+        path: ${{parent.jobs.sampler_fewshot.outputs.output_dataset}}
+      template_input: "{\n  \"activity_label\": {{activity_label}},\n  {% for i in\
+        \ range(endings|length) %}\n    \"choice_{{(i+1)}}\": {{endings[i]}},\n  {%\
+        \ endfor %}\n  \"ctx\": {{ctx}},\n  {% set label_int = label | int %}\n  \"\
+        label\": {{(label_int + 1)}}\n}"
+    outputs:
+      output_dataset:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.jsonl
+  promptcrafter:
+    type: command
+    component: azureml://registries/azureml/components/prompt_crafter/labels/latest
+    limits: {}
+    inputs:
+      test_data:
+        type: uri_folder
+        path: ${{parent.jobs.preprocessor.outputs.output_dataset}}
+      few_shot_data:
+        type: uri_file
+        path: ${{parent.jobs.preprocessor_fewshot.outputs.output_dataset}}
+      prompt_type: chat
+      prompt_pattern: 'Category: {{activity_label}}
+
+        Text: {{ctx}}
+
+        Completion options:
+
+        (1) {{choice_1}}
+
+        (2) {{choice_2}}
+
+        (3) {{choice_3}}
+
+        (4) {{choice_4}}
+
+        The most likely text completion is: '
+      n_shots: 5
+      output_pattern: '{{label}}'
+      system_message: Given a partial description of an event as text, your task is
+        to select the most appropriate continuation from the provided options. Carefully
+        analyze the context and use your understanding of the world to make the best
+        choice. Consider logical flow, relevance, and commonsense reasoning when making
+        your selection. Always choose the answer from provided completion options
+        that best completes the given sentence or scenario.
+      random_seed: 0
+    outputs:
+      output_file:
+        type: uri_file
+        path: azureml://datastores/${{default_datastore}}/paths/azureml/${{name}}/${{output_name}}.jsonl
+  batch_score_preprocessor:
```

Comment:
Instead of using these components separately, we should use the pipeline component: `batch_benchmark_inference_with_inference_compute`. This would also simplify the pipeline at UI.

Reply:
With this component Im getting the following error - Specified image azureml/curated/model-evaluation reached EOL
So I have pushed a new version of the component with updated model-evaluation version